### DISCUSSION TITLE
Added "mapping participants" displayed on the song selection bar.

### DISCUSSION THREAD

**Discussion by huyenden** [ppy/osu#24466] [Ideas] [answered]
The story is that when I click on the quick link that leads to the person 'map maker', it shows the name 'mapping participant'.
_Although the HUD only shows the default 'map creator', it doesn't show that 'mapping participant'._

So I think if it is possible to add the name of the 'mapping participant' next to the 'map creator', it will be easier to understand and intuitive, it will be easier for players to recognize which quick links lead to the profiles of these player. Instead of having to find out in the beatmap details.

My brief description is like the video below _(Opss 10MB)_:

https://github.com/ppy/osu/assets/128424762/0a5987f4-9f02-418b-9453-10d4f98bd921

Or a little idea with lines like this:
![image](https://github.com/ppy/osu/assets/128424762/d90b3800-b369-4ec8-98ee-6d2dd47a53e6)

**Comment by nagi-desuuu** [comment, marked as answer]
Related to (if not Duplicate of) https://github.com/ppy/osu/issues/24143

**Reply by huyenden** [reply]
So maybe this is still being updated, thanks for reminding me
